tests/unit/keyinput/test_basekeyparser.py

```diff
@@ -215,14 +215,14 @@ def test_number_press(self, fake_keyevent, keyparser,
 
     @pytest.mark.parametrize('modifiers, text', [
         (Qt.NoModifier, '2'),
-        (Qt.KeypadModifier, 'num-2'),
+        (Qt.KeypadModifier, '2'),
     ])
     def test_number_press_keypad(self, fake_keyevent, keyparser, config_stub,
                                  modifiers, text):
-        """Make sure a <Num+2> binding overrides the 2 binding."""
+        """Make sure a <Num+2> binding yields the 2 binding."""
         config_stub.val.bindings.commands = {'normal': {
             '2': 'message-info 2',
-            '<Num+2>': 'message-info num-2'}}
+            '<Num+2>': 'message-info 2'}}
```

Comment:
Those changes seem incorrect - a `num-2` binding should still take precedence over a `2` binding (as `is_special` is only used for hints and marks).